Source/DateTimePicker.swift

```diff
@@ -260,6 +266,9 @@ public protocol DateTimePickerDelegate: AnyObject {
             timeZone = calendar.timeZone
         }
     }
+    
+    public var autoSelectDayOnStopScroll: Bool = true
```

Comment:
I'm not quite fond of this feature, as although someone requested it I find it looks buggy and not what I intended for the library. I think it's fine if someone wants it for their app, they can maybe tweak it to their need; but I'd prefer to leave it off the library for now. WDYT?